ledger/handle/get_code.go

```diff
@@ -0,0 +1,65 @@
+///
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+///
+
+package handle
+
+import (
+	"context"
+
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/flow"
+	"github.com/insolar/insolar/insolar/flow/bus"
+	"github.com/insolar/insolar/insolar/message"
+	"github.com/insolar/insolar/insolar/reply"
+	"github.com/insolar/insolar/ledger/proc"
+	"github.com/pkg/errors"
+)
+
+type GetCode struct {
+	dep *proc.Dependencies
+
+	Message bus.Message
+}
+
+func (s *GetCode) Present(ctx context.Context, f flow.Flow) error {
+	msg := s.Message.Parcel.Message().(*message.GetCode)
+
+	jet := s.dep.FetchJet(&proc.FetchJet{Parcel: s.Message.Parcel})
```

Comment:
Could you please explain why this code is equivalent to jetFromContext used before?